src/planwise/client/projects/components/sidebar.cljs

```diff
@@ -1,15 +1,35 @@
 (ns planwise.client.projects.components.sidebar
+  (:require-macros [reagent.ratom :refer [reaction]])
   (:require [re-frame.core :refer [subscribe dispatch]]
             [re-com.core :as rc]
             [planwise.client.components.progress-bar :as progress-bar]
             [planwise.client.components.filters :as filters]
-            [planwise.client.projects.db :as db]))
+            [planwise.client.projects.db :as db]
+            [planwise.client.utils :as utils]))
 
 
+(defn- demographic-stat [title value]
+  [:div.stat
+   [:div.stat-title title]
+   [:div.stat-value value]])
+
 (defn- demographics-filters []
-  [:div.sidebar-filters
-   [:div.filter-info
-    [:p "Filter here the population you are analyzing."]]])
+  (let [current-project (subscribe [:projects/current-data])
+        population (reaction (:region-population @current-project))
+        area (reaction (:region-area-km2 @current-project))
+        density (reaction (/ @population @area))]
```

Comment:
These 3 don't need to be a reaction, since they will not be changing independently of the current project. Remove the `reaction`, and remove the deref (ie the `@`) from their uses in the code.